Antidote/ActiveCallViewController.m

```diff
@@ -93,4 +187,55 @@ - (NSString *)stringFromTimeInterval:(NSTimeInterval)interval
 
     return [NSString stringWithFormat:@"%02d:%02d", minutes, seconds];
 }
+
+- (void)toggleMicrophone
+{
+    if (self.manager.enableMicrophone) {
+        self.manager.enableMicrophone = NO;
+
+        UIImage *image = [UIImage imageNamed:@"call-microphone-disable"];
+        [self.microphoneButton setImage:image forState:UIControlStateNormal];
+    }
+    else {
+        self.manager.enableMicrophone = YES;
+        UIImage *image = [UIImage imageNamed:@"call-microphone-enable"];
+        [self.microphoneButton setImage:image forState:UIControlStateNormal];
```

Comment:
It would be better to set one image for Normal state of button, another for Selected (this should be done in `create...` method).

Here you would need only to change `selected` state of button.
```
- (void)toggleMicrophone
{
    BOOL enabled = ! self.manager.enableMicrophone;
    self.manager.enableMicrophone = enabled;
    self.microphoneButton.selected = enabled;
}
```

Same for mute button.